### DISCUSSION TITLE
为什么图片上传之后会变糊而且大小还变大了？Why does the image become blurry and the size increases after uploading?

### DISCUSSION THREAD

**Discussion by a233a2** [CaiJimmy/hugo-theme-stack#1100] [Q&A]
这是同一张图片在本地和从我网站上下载下来的，为什么感觉出现了撕裂感？    
This is the same image downloaded locally and from my website, why does it feel torn?    
![QQ_1732866128446](https://github.com/user-attachments/assets/7ae031f2-be86-4b91-9a97-d3ac6d8a0907)

**Comment by a233a2** [comment]
我把两个图像处理的功能全部关闭了。一个是cover的一个是content的 图像处理全部关闭了 。Action部署了好长时间后图片就清晰了。         
之前专门下载了已部署到网站的图像发现与本地图像相比 两种清晰度的图片的大小是一样的。那为什么还需要这个功能？

**Reply by warrenwang10k** [reply]
与模板无关，本质上是Hugo提供的图片处理的默认配置（75%质量）牺牲了画质（在原图片的压缩算法优秀的情况下，它有大概率也不能得到更小文件大小）。

建议关闭content，保留cover。因为cover会用于 相关文章，存档列表等封面图（这种情况下图片尺寸较小，120~250px），处理图片大小后，不加载原图，能得到更快的加载速度
